src/emscripten/libUncrustify.d.ts

```diff
@@ -0,0 +1,823 @@
+interface EmscriptenVector< T >
```

Comment:
This is a declaration file for Typescript. It helps working with the generated Emscripten library in a Typescrript environment. 

It is was also created by hand and the enum values should also be generated at some point.